doc/GettingStarted.md

```diff
@@ -5,81 +5,24 @@ uid: Uno.Extensions.HowToGettingStarted
 
 This tutorial will walk through how to create an Uno application with the `dotnet new` tool, which is already configured to use the Uno.Extensions.
```

Comment:
I think we need to adjust here, this is not about how to create an Uno application anymore, but about exploring the solution and how to run the app.

This could be too deep, but do we really need this page? Wouldn't be enough to just point people to how to setup/create an app / debug docs on the ExtensionsOverview.md page?

@dansiegel @jeromelaban @agneszitte

Reply:
It's indeed not particularly useful now, as we cover it in the general get started. We can remove it as well as in the toc.

Reply:
@jeromelaban Done. The commit https://github.com/unoplatform/uno.extensions/pull/2249/commits/7362080df46a2221dd2bc4058ffa60ea5677d989 removes _How-To: Creating an application with Uno.Extensions_, its images and replaces its references with the [Creating an app](https://platform.uno/docs/articles/create-an-app-vs2022.html?tabs=windows) docs.